posts/2019-10-1-serverless-azure-functions-v1-part2.md

```diff
@@ -0,0 +1,337 @@
+---
+title: "How to Create a REST API with Azure Functions and the Serverless Framework - Part 2"
+description: "Learn how to add API Management, Webpack and CI/CD to your Azure Functions REST API"
+date: 2019-10-1
+thumbnail: "https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/azure-functions-part1/thumbnail.png"
+heroImage: "https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/azure-functions-part1/header.png"
+category:
+  - guides-and-tutorials
+authors:
+  - TannerBarlow
+---
+
+#### Overview
+
+Now that you've created and deployed a basic API from [Part 1](https://serverless.com/blog/serverless-azure-functions-v1), let's take a few more steps towards making that API more resilient and secure. This post will still be based on the [example repo](https://github.com/tbarlow12/sls-az-func-rest-api), and will follow the same "commit-per-step" format as [Part 1](https://serverless.com/blog/serverless-azure-functions-v1), which contains Steps 1 and 2.
+
+#### [Step 3: Add unit testing and linting](https://github.com/tbarlow12/sls-az-func-rest-api/commit/465ecfe04bda8d4d5ac7c9c5ce31557a8993408f)
+
+Because this isn't a blog post on unit tests, linting or quality gates in general, I'll just share the tools that I'm using and the quality gates that I added to the repository. Feel free to use them as stubs for your own future tests or lint rules.
+
+For unit tests, I'm using the [Jest](https://jestjs.io/) test runner from Facebook. I've used it for several projects in the past and have never had any issues. Because my code makes REST calls via `axios`, I'm using the `axios-mock-adapter` to mock the request & response. Jest tests typically sit alongside the file they are testing, and end with `.test.js`. For linting, I'm using [ESLint](https://eslint.org) with a very basic configuration.
+
+Check out the [commit in the example repo](https://github.com/tbarlow12/sls-az-func-rest-api/commit/465ecfe04bda8d4d5ac7c9c5ce31557a8993408f) for more details.
+
+#### [Step 4: Add basic API Management Configuration](https://github.com/tbarlow12/sls-az-func-rest-api/commit/c593308efc5a60e2701ec97122564592072080e2)
```

Comment:
For Step 4, 5 and 5.1 I don't see instructions to run any commands?

If they need to run `sls deploy` in order to make that change lets remind them. But more importantly if there is some way to demonstrate the change that we just made let's do that. e.g. - Can we show them that auth is now required for the API after a deploy? That's super cool if we can add it.